dashboards/ds-solarwinds-webapi/dashboard.json

```diff
@@ -0,0 +1,418 @@
+                            "template": "{{#if value == \"0\" }}<img src=\"/images/health-icons/unknown.png\" height=\"16\"/>{{elseif value == \"1\" }}<img src=\"/images/health-icons/up.png\" height=\"16\"/>{{elseif value == \"2\" }}<img src=\"/images/health-icons/down.png\" height=\"16\"/>{{elseif value == \"3\" }}<img src=\"/images/health-icons/warning.png\" height=\"16\"/>{{elseif value == \"14\" }}<img src=\"/images/health-icons/critical.png\" height=\"16\"/>{{else}}<img src=\"/images/health-icons/unknown.png\" height=\"16\"/>{{/if}}"
+                          },
+                          "SysName": {
+                            "displayName": "Node"
+                          },
+                          "Vendor": {
+                            "displayName": "Vendor"
+                          },
+                          "Description": {
+                            "displayName": "Description"
+                          },
+                          "Location": {
+                            "displayName": "Location"
+                          },
+                          "CPUCount": {
+                            "displayName": "# CPU"
+                          },
+                          "IPAddress": {
+                            "displayName": "IP Address"
+                          },
+                          "AvgResponseTime": {
+                            "displayName": "Response Time",
+                            "template": "{{value}} ms"
+                          },
+                          "LastBoot": {
+                            "displayName": "Last Boot",
+                            "template": "{{timeago(value)}}"
+                          },
+                          "StatusDescription": {
+                            "displayName": "Summary",
+                            "template": "{{value.substr(0,100)}}"
+                          }
+                        },
+                        "customColumns": {},
+                        "columns": [
+                          "Status",
+                          "SysName",
+                          "Vendor",
+                          "Description",
+                          "Location",
+                          "CPUCount",
+                          "IPAddress",
+                          "AvgResponseTime",
+                          "LastBoot",
+                          "StatusDescription"
+                        ],
+                        "autohide": false,
+                        "rowLink": "http://int-solarwinds.int.squaredup.com:8787/Orion/NetPerfMon/NodeDetails.aspx?NetObject=N:{{NodeID}}"
+                      }
+                    },
+                    "description": "",
+                    "title": "All Nodes"
+                  }
+                ]
+              }
+            ],
+            "width": 75.146
+          }
+        ]
+      }
+    ]
+  },
+  "derivedFrom": "00000000-0000-0000-0000-000000000000",
+  "draft": true,
```

Comment:
I usually delete the draft line. Also I think this means this dashboard will be imported as a draft unpublished. Maybe @adamkinniburgh can confirm

Reply:
Disasterous for dashboard packs, but in this case it shouldn't hurt as once you've pasted in the JSON and clicked apply, you'd then click Publish regardless, and the `draft: true` would be dropped to false as part of that.